### PR TITLE
add a command to obtain a current log level of qtile.

### PR DESCRIPTION
**Pull request by sux2mfgj**
I also add a test for related commands.

### CODE REVIEW

File: libqtile/manager.py [hunk 1]
```diff
@@ -1196,6 +1196,9 @@ def cmd_critical(self):
         logger.setLevel(logging.CRITICAL)
         logger.critical('Switching to CRITICAL threshold')
 
+    def cmd_loglevel(self):
+        return (logging.getLevelName(logger.level), logger.level)
```
Comment (ramnes) [inline libqtile/manager.py]: Why do we need both? Can't we just have enough with one?

File: test/test_manager.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
 import pytest
 import subprocess
 import time
+import logging
```
[the remark this answers is not in the document]
Comment (sux2mfgj) [reply test/test_manager.py]: ok. I fixed it with a new commit.

File: libqtile/manager.py (hunk 1 above)
Comment (sux2mfgj) [reply libqtile/manager.py]: The logger.level is represented by numerical values. [a link of the reference](https://docs.python.org/3/howto/logging.html#logging-levels).
It is difficult to understand for me, so I wrote both. Is it better a idea to remove the left one (loggin.getLevelName(logger.level))?

Comment (ramnes) [reply libqtile/manager.py]: What would you think of doing two commands, `cmd_loglevel` and `cmd_loglevelname`?

File: test/test_manager.py [hunk 3]
```diff
@@ -39,6 +39,8 @@
 import libqtile.hook
 import libqtile.confreader
 
+import logging
```
Comment (ramnes) [inline test/test_manager.py]: Haha, it should have been ordered alphabetically but still in the first block, as it is an import from the standard library. I should have mentioned it, sorry! But honestly that is no big deal, I'll take that on myself if you want, as we don't have any clear rule stated so far anyway.

File: libqtile/manager.py (hunk 1 above)
Comment (sux2mfgj) [reply libqtile/manager.py]: I think it is good. I'll make a commit.

File: test/test_manager.py (hunk 3 above)
Comment (sux2mfgj) [reply test/test_manager.py]: oh.. I'm sorry the simple my mistake. The rule is mentioned in pep8. Usually, Qtile developers are following that rule, I think.

Comment (sux2mfgj) [reply test/test_manager.py]: I fixed it!

File: libqtile/manager.py (hunk 1 above)
Comment (sux2mfgj) [reply libqtile/manager.py]: Sorry to bother you again. I changed the test.

**Comment by ramnes** [comment]
Thank you!

**Pull request merged**